spec.html

```diff
@@ -541,7 +541,7 @@ <h1>The Syntactic Grammar</h1>
       </emu-note>
       <emu-note>Parse Nodes are specification artefacts, and implementations are not required to use an analogous data structure.</emu-note>
       <p>Productions of the syntactic grammar are distinguished by having just one colon &ldquo;<b>:</b>&rdquo; as punctuation.</p>
-      <p>The syntactic grammar as presented in clauses 12, 13, 14 and 15 is not a complete account of which token sequences are accepted as a correct ECMAScript |Script| or |Module|. Certain additional token sequences are also accepted, namely, those that would be described by the grammar if only semicolons were added to the sequence in certain places (such as before line terminator characters). Furthermore, certain token sequences that are described by the grammar are not considered acceptable if a line terminator character appears in certain &ldquo;awkward&rdquo; places.</p>
+      <p>The syntactic grammar as presented in clauses <emu-xref href="#sec-ecmascript-language-expressions"></emu-xref> through <emu-xref href="#sec-ecmascript-language-scripts-and-modules"></emu-xref> is not a complete account of which token sequences are accepted as a correct ECMAScript |Script| or |Module|. Certain additional token sequences are also accepted, namely, those that would be described by the grammar if only semicolons were added to the sequence in certain places (such as before line terminator characters). Furthermore, certain token sequences that are described by the grammar are not considered acceptable if a line terminator character appears in certain &ldquo;awkward&rdquo; places.</p>
```

Comment:
I'd rather list them out instead of using a range. Now we're dependent on the clause ordering. If we list them out, we can re-order clauses all we want without breaking these references.

Reply:
@michaelficarra We'd still need to update this sentence because such a reordering would result in these being listed out of order.

More generally, I think readability ought to take precedence over maintainability (especially for things which rarely require maintenance, as here), and I think "as presented in clause 12 through 15" is more readable than "as presented in clauses 12, 13, 14, and 15", so I would prefer to keep the current PR's wording.

Reply:
I will reluctantly accept this change since it is easy and doesn't make things worse. For posterity, my preference would be to list out an `<emu-xref>` for each clause, and when ecmarkup sees that they represent a range, it could render it as we see here. This would address the issues of finding occurrences of the clause ID within the source and including the reference in a clause's reference list. But the other editors don't agree it's worthwhile.